sklearn/linear_model/bayes.py

```diff
@@ -453,6 +453,9 @@ class ARDRegression(RegressorMixin, LinearModel):
     scores_ : float
         if computed, value of the objective function (to be maximized)
 
+    intercept_ : float
+        Independent term in decision function. Set to 0.0 if ``fit_intercept = False``
```

Comment:
See flake8 error output:
 - line longer than 80 char (a line break is needed)
 - the below empty lines should not contain spaces